ext/oj/oj.h

```diff
@@ -153,6 +153,8 @@ typedef struct _Options {
     char		create_ok;	// YesNo allow create_id
     char		allow_nan;	// YEsyNo for parsing only
     char		trace;		// YesNo
+    int64_t     integer_range_min; // dump numbers outside range as string
```

Comment:
I hate to be picky but please line these up with the rest of the names. Keep the code looking nice.

Reply:
Fixed